titles/aap-on-gcp/aap-on-gcp.html

```diff
@@ -4,7 +4,7 @@
 <meta charset="UTF-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="generator" content="Asciidoctor 2.0.17">
```

Comment:
I don't believe we're supposed to deliver the generated file, but the experts can clarify.

Reply:
It doesn't matter: once a PR is merged, a GitHub action regenerates the HTML file.